Week4/homework/ex1-aggregation/ex2/transfare.js

```diff
@@ -0,0 +1,41 @@
+async function transfer(client, accountsCollection, accountA, accountB, amount, remark) {
```

Comment:
To make it a bit more clear you should refrain from using `paramA`, `paramB`.

```suggestion
async function transfer(client, accountsCollection, fromAccount, toAccount, amount, remark) {
```